### PR TITLE
#161883389 Build the update feature where you can update status

### PR DESCRIPTION
**Pull request by Bakley**
### What does this PR do?
Build the PUT method to update status

### Description of Task to be completed?
Has the following API Endpoints working:

`PUT api/v1/parcel/<id>/cancel`

### How should this be manually tested?
After cloning the repo, setting up virtualenv and installing the required requirements
RUN
`export FLASK_ENV=development`
`export FLASK_APP=run.py`
`flask run`

Using POSTMAN test the above endpoints
### What are the relevant pivotal tracker stories?
#161883389

### Screenshots (if appropriate)
cancel_api

### CODE REVIEW

**Comment by cosmas28** [comment]
Kindly fix the failing tests.

File: app/__init__.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+
+def create_app(config_name):
+    """Create the app with the desired environment."""
+    # instantiate the flask app
+    app = Flask(__name__)
+    app.url_map.strict_slashes = False
+    # app setting config
+    app.config.from_object(app_config[config_name])
+    # app.config.from_pyfile('config.py')
```
Comment (cosmas28) [inline app/__init__.py:16]: Kindly remove the commented codes which have no use here.

File: config.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+class Production(Config):
+    DEBUG = False
+    TESTING = False
+
+class Testing(Config):
+    DEBUG = True
+    TESTING = True
+
+app_config = {
+    "development" : Development,
```
Comment (cosmas28) [inline config.py:22]: Kindly use either single or double quotes throughout your code. Don't mix them.

**Review by cosmas28 — CHANGES_REQUESTED** [review]

File: app/__init__.py (hunk 1 above)
Comment (Bakley) [reply app/__init__.py:16]: Okay, all done

File: config.py (hunk 2 above)
Comment (Bakley) [reply config.py:22]: Thank you all resolved

**Pull request merged**